forecast-admin/forecast/opportunities/management/commands/load_opportunities.py

```diff
@@ -0,0 +1,193 @@
+import os
+import csv
+from datetime import date
+from decimal import Decimal, InvalidOperation
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ValidationError
+from django.db import transaction
+from opportunities.models import Office, Opportunity, OSBUAdvisor
+from optparse import make_option
+
+
+# A data loader of contract opportunities
+# Thank you Brendan Sudol... :)
+# https://github.com/brendansudol/calc/blob/ca4a13d0fa34a468d788710975bf71959eb8bb20/contracts/management/commands/load_s70.py
+class Command(BaseCommand):
+    help = 'Load data from CSV'
+
+    default_filename = 'forecast/fixtures/fy16q1.csv'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '-f', '--filename',
+            default=default_filename,
+            help='input filename (.csv, default {})'.format(
+                default_filename
+            )
+        ),
+        make_option(
+            '-a', '--append',
+            dest='replace',
+            action='store_false',
+            default=True,
+            help='Append data (default is to replace).'
+        ),
+        make_option(
+            '-s', '--strict',
+            action='store_true',
+            default=False,
+            help='Abort if any input data fails validation.'
+        ),
+    )
+
+    def handle(self, *args, **options):
+        filename = options['filename']
+        if not filename or not os.path.exists(filename):
+            raise ValueError('invalid filename')
+
+        OpportunitiesLoader().load(
+            filename,
+            replace=options['replace'],
+            strict=options['strict']
+        )
+
+
+class OpportunitiesLoader(object):
+    model = Opportunity
+    header_rows = 1
```

Comment:
You don't need this - because there's no way to change this on the command line.